app/scripts/disable-console.js

```diff
@@ -0,0 +1,9 @@
+// Disable console.log in contentscript to prevent SES/lockdown logging to external page
+// eslint-disable-next-line import/unambiguous
+if (
+  !(typeof process !== 'undefined' && process.env?.METAMASK_DEBUG) &&
```

Comment:
In testing, I discovered that this does not work. It's not being replaced correctly by `loose-envify`. It's broken by the optional chaining operator - `loose-envify` requires that it be _exactly_ `process.env.___`. e.g.:
```suggestion
  !(typeof process !== 'undefined' && process.env.METAMASK_DEBUG) &&
```